hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/types/NSSummary.java

```diff
@@ -20,51 +20,67 @@
 
 import org.apache.hadoop.ozone.recon.ReconConstants;
 
+import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.List;
 
 /**
  * Class to encapsulate namespace metadata summaries from OM.
  */
 
 public class NSSummary {
   private int numOfFiles;
-  private int sizeOfFiles;
+  private long sizeOfFiles;
   private int[] fileSizeBucket;
+  private List<Long> childDir;
```

Comment:
Just a note: If we are implementing `appendToChildDir`/`removeFromChildDir`, maybe use `Set<>` here instead?

Reply:
Yeah makes sense, but maybe on the next patch? otherwise there'll be more conflicts to resolve.